tests/calculus-endpoints/twitter-user-credibility-without-twitter-api.ts

```diff
@@ -0,0 +1,35 @@
+import request from 'supertest'
+import app from '../../src/app'
+import { Credibility, TwitterUser } from '../../src/calculator/models'
+
+interface userCredibilityParams extends TwitterUser {
+    verified: boolean,
+    yearJoined: number
+}
+
+describe('/calculate/user/scrape/:verified/:accountCreationYear endpoint', () => {
```

Comment:
the endpoint is not correct here, it doesn't affect anything but would be great if you update it to avoid confusions in the future.

Reply:
done